prime-router/metadata/hl7_mapping/ORU_R01/base/observation-request.yml

```diff
@@ -13,17 +13,17 @@ elements:
     schema: ../../common/organization-identifier
 
   - name: observation-request-date-time
-    condition: '%resource.effective is dateTime'
+    condition: '%resource.effective.exists() and %resource.effective.is(dateTime)'
```

Comment:
Almost all the changes boil down to a change in how most FhirPathEngine functions get evaluated.  Previously, something like `is` would return a boolean when the focus resource is empty, now it will actually return an empty array which can't be used as a condition